hudi-hadoop-common/src/test/java/org/apache/hudi/metadata/TestHoodieTableMetadataUtil.java

```diff
@@ -856,4 +856,260 @@ private static Stream<Arguments> mapKeyNoSeparatorToFileGroupIndexTestCases() {
         )
     );
   }
+
+  /*
+  @Disabled
+  @Test
+  public void testConvertMetadataToPartitionStatRecordsWithoutColumnStats() throws Exception {
```

Comment:
why disabled?

Reply:
Removed since these are temporary tests before the util method is moved to another class.